### ISSUE TITLE
Clinical pathology reports as phenotype

### ISSUE THREAD

**Issue by selewis** [monarch-initiative/monarch-legacy#584]
I think it would be very helpful to extend HPO to include more terms/synonyms that are commonly used in clinical pathology reports. For example, diabetes mellitus includes hyperglycemia (HP) as one of its phenotypes, which is great, but "high serum glucose measurement", "glucose challenge low" (or high), are not included as related/relevant synonyms. For mining EHRs (obtaining data from EHRs), and for people searching for things it would be useful to add more of these. 

More confusion. Can find both "increased circulating glucose level" (MP) and "high blood glucose levels" (MP) and "Increased Blood Glucose Level" (HP synonym of hyperglycemia). No graph view makes it easy to chase around in circles (while leaving breadcrumbs on a piece of note paper nearby)

**Comment by iamkoehler** [comment]
Hi,

> I think it would be very helpful to extend HPO to include more terms/synonyms that are commonly used in clinical pathology reports. For example, diabetes mellitus includes hyperglycemia (HP) as one of its phenotypes, which is great, but "high serum glucose measurement", "glucose challenge low" (or high), are not included as related/relevant synonyms. For mining EHRs (obtaining data from EHRs), and for people searching for things it would be useful to add more of these.

I am happy to add more classes to HPO if you send me a list. You can also use the sourceforge tracker. Last but not least there is a phenotype-editors-list (phenotype-ontologies-editors@googlegroups.com), where MP and HP are coordinating their efforts.

> More confusion. Can find both "increased circulating glucose level" (MP) and "high blood glucose levels" (MP) and "Increased Blood Glucose Level" (HP synonym of hyperglycemia). No graph view makes it easy to chase around in circles (while leaving breadcrumbs on a piece of note paper nearby)

MP seems to make a distinction between here between an increased form and a pathologically increased form. But this is definitely something for the phenotype-editors-list.
I think you are right in that there is no dedicated graphview tool for this. 

Best,
Sebastian=

**Comment by selewis** [comment]
I'll look for a list of standard clinical report terms. I tried yesterday,
but wasn't successful after ~15 minutes, apparently it might be a hospital
to hospital "standard".

-S

**Comment by pnrobinson** [comment]
This is not a particular problem, and if there are standard phrases, they can be entered as SYNONYM or whatever is appropriate. What though is the use case? Are the pathology reports going to be text mined?
-Peter

**Comment by nlwashington** [comment]
I’ve been told by a certain someone that there’s lots of terms that can be mined out of http://mtsamples.com/ to flesh out synonyms and create new terms.  Is there a student or postdoc that could work on that?

**Comment by selewis** [comment]
Yes, apparently some groups are mining EHR with these, thinking that
focusing solely on the pathology reports might be most useful in
interpretation GWAS-y&PheWas-y.

**Comment by pnrobinson** [comment]
Can we get a list of these synonyms
-Peter

**Comment by selewis** [comment]
Unfortunately it was just through reading a paper (i.e. they didn't offer
much and it would be manual). What I was wondering is whether there are
some/any sort of existing standards for pathologists we might pull from and
then do the usual text-mining+manual extensions. Maybe not worth the
effort, it simply occurred to me when I was reading the paper and then used
the text from one of the examples in a figure to search monarch and got
zero hits (It was something about glucose levels).

-S
